### ISSUE TITLE
use single color to render the back side of terrain

### ISSUE THREAD

**Issue by vinstics** [CesiumGS/cesium#8942]
Thanks for improving the underground navigation experience. It's important when dealing with some underground project. But sometimes it's hard to distinguish if the current view point is underground because the terrain is rendered the same on both sides. Maybe we can make it more clear by using single color to render the back side of terrain.

**Comment by vinstics** [comment]
@OmarShehata It maybe something like this.
![image](https://user-images.githubusercontent.com/16721670/84355919-00167e00-abf6-11ea-9e2f-93b3858012b6.png)

![image](https://user-images.githubusercontent.com/16721670/84356541-e6296b00-abf6-11ea-954d-1367d88f47ae.png)

![image](https://user-images.githubusercontent.com/16721670/84356618-048f6680-abf7-11ea-835c-1384d8299115.png)

**Comment by lilleyse** [comment]
@vinstics you might be able to use `globe.undergroundColor` for this. See this [sandcastle demo](https://sandcastle.cesium.com/#c=vVVNbxoxEP0rFqelAi+UJFISEpVASCO1BAXSqio9mN2BtTD2yvaS0Kr/vWObJUASqeqhHCzG8+k3b2bjmDwYIIngec7lnOSCSTDEKmJAQGL5CsSaZDwFkjNtDVEzYjMgc6GmQEyhZywBOpETuWKarDg8giYXRMIj6YLhxZJ+8XfRpJJ4uaukZVyCnlRq5NdEEmIW645dKpNnoOGMzJgwUHMKC1qj6VCrFebXZ2XERAOz8FVpkY6DSVRFh9/V87IOk4AELCPUQ714HlSh8ItgQr3k3YJsMDbIUY6P6rIlaObK1UoIjAKSTQV0UeCGK9kD6/BREoP5ms/LIDM13xinOzqfippMPd5oVcj0+c0vjKYsWfRdCQXmwqa8FmS04K4dhyoMDHruE2ClyvVig5oX6d19Z3Bz/aZ5R+QZu1r3uLFMJrDfyQEw3Wd6lDDBdNSgjRpp0GN3hH8B/zgmQ54sCCO5MtwDxKznzI1mMiVdJtdq06etxXOVSDI11yzPeEKtciIqmIwcIXZq2bObabXswRxbZ6J6s9mi709Ojo9bRzXSOqGN09Zp6wjLbDYa1YmsbniQPj+x1WjgEzb3U4cFgj4qW7OT9GpPF5XV17bBAgIB2nKkhmGi9gJ1d3WOUYFJkZ+HpUpBfGZW86ct5ccInZkpvTQUmLEDpW32kI9Vnz9B2tdI1W01VT87YY7PyHcnkDdzR0H9AlqPeStqhs6GIwR2v/K5QS7v/zFH/X8k8eGb/yFH/e+S/PBnIbFfZYp7mKOELbO6CPtvs0O2N+WCC0st8fuJOumAl853n8a1g+n5CMwph9wm2T2Tc4j8INf9ue9KNZoWhrwjx3Q7P3sVCKUWHbtlaLTJERh8RG9714Px7fibc6zUKm1j1wIuS3A+8GWOZCaFFhGlsYUlMhcRjadFsgBLE2OcozNtx7uu7ZSvCE8vXvmu4NeMGYOaGe7PEf8Jk8plO0b7F65CeRzuVqAFWzuzrHn5KVxSStsxiq97WqXElOmDyH8A)

![underground-color](https://user-images.githubusercontent.com/915398/84393169-3ee20f00-abc9-11ea-9cb8-8af6f13d14c4.gif)

**Comment by vinstics** [comment]
@lilleyse @OmarShehata 
Thanks a lot. Looks good enough. It keeps the shadow of the terrain without redering by using alpha. However, there is still room for improvement when use it together with `globe.translucency`. 
Something like `undergroundColorGradientByDistance` might help, which can blend backside of the terrain with some color you want while keeping the center of the globe remain black.

underground
![image](https://user-images.githubusercontent.com/16721670/84458146-e8430680-ac96-11ea-9cc9-aa22bb621dd3.png)

above the terrain
![image](https://user-images.githubusercontent.com/16721670/84458304-4cfe6100-ac97-11ea-9860-0e6e937b0036.png)

**Comment by lilleyse** [comment]
@vinstics yeah I see, the current system only supports one color but for your case you would need two.

As a partial workaround you could check when `scene.cameraUnderground` is true or not and change the color based on that: [Sandcastle example](https://sandcastle.cesium.com/#c=pVRtT9swEP4rp35Kp+K0K0UCClopjKExQOVlmtQvbnJpLRw7sp1CN/Hfd3bS0jI2se1LpMvdc/fc3XOOY7i1CIkURSHUFArJFVpwGixKTJyYo1zATKQIBTfOgs7AzRCmUk8QbGkyniAbq7GacwNzgQ9o4AAUPsAQrShzdhf+ReNGEuyhVo4LhWbcaMGPsQKw94uBy7UtZmhwDzIuLba8w6ExFHpl9Jzqm71lxsQgd/hVG5neVCFRkwBPzf0lD5ugQqJR8WHB3K9cFfGDKoQFq/agpCFYLVLyhv9s9aeOyIU6FtZxlfgMKy/L+aPIy3zEU1FaiKHTbrdZewnij28C9QJirCpilppEdV3QdIc8R8P93IyWktpBxScSh2QIK7Q6RucXpRWlD8OjNBV/Z7iysqSEyaJG+eacKX8Tkxmq8pFqDmQx4xTaZr1AKo7hSiT3wKHQVoRi3AUhnBI+hSFXC10PfxVxsFzYkJSjp4YXM5Ewp71JDq4iv+U1qWzEEZf8GKc0BhttdTpd9n5np9frbregu8Pau93d7nav5UfdHKtmPev0edDd9R1MdKlSkvd10NimPo82fNGSfWuVrJZVJaUkLIN56wXQ97JZp/WivU/IvfNKuGQ24mqKEY23BVvhuwllppLFOy+LZYMbDKTW9wN347eXaZNHdY0v3BnxuM3Ojk8ubs5uvjXXNEWtuREqOiXG0/RkjsqdU4/kMlFWqqChqFkdpcggqlBVvVsPmxpPso6A+kjKZw9JUpu1tXuTXY4GF6cn+3+CBLUdLZZnwhRyc8dliSsFvh2brUPbAfpEV0dv3F+xPjofDD//B+n2v5LuLEmHB63RavStW0g8rNIBfBB5oY2D0siIsdhhTk82nVM8KZN7dCyxtlnX7sfr0H4q5iDSg1ceYnr+ubXkyUopr8V3HDcO+zHF/wKVOkj4co5G8oUPm3UOz6ufjLF+TObrSKe1nHDzIvNP)
